DigitalLearningSolutions.Data/Models/SectionContent/SectionContent.cs

```diff
@@ -1,30 +1,59 @@
 ﻿namespace DigitalLearningSolutions.Data.Models.SectionContent
 {
+    using System.Collections.Generic;
+
     public class SectionContent
     {
         public string CourseTitle { get; }
         public string SectionName { get; }
-        public int? TimeMins { get; }
+        public int TimeMins { get; }
         public int AverageSectionTime { get; }
         public bool HasLearning { get; }
         public double PercentComplete { get; }
+        public int DiagAttempts { get; }
+        public int SecScore { get; }
+        public int SecOutOf { get; }
+        public string DiagAssessPath { get; }
+        public string PLAssessPath { get; }
+        public int AttemptsPL { get; }
+        public int PLPassed { get; }
+        public bool DiagStatus { get; }
+        public bool IsAssessed { get; }
```

Comment:
We don't need to stick to the same names as in the old system, so I think some of these names could be improved 🙂